src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -292,16 +278,39 @@ export class IncidentDetail extends React.Component {
 
 IncidentDetail.defaultProps = {
   previewState: '',
-  attachment: '',
+  attachmentHref: '',
 };
 
 IncidentDetail.propTypes = {
   previewState: PropTypes.string,
-  attachment: PropTypes.string,
-
-  incidentModel: PropTypes.object.isRequired,
-  historyModel: PropTypes.object.isRequired,
-  categories: PropTypes.object.isRequired,
+  attachmentHref: PropTypes.string,
+
+  incidentModel: PropTypes.shape({
```

Comment:
`incidentType`?

Reply:
het incident model is een child van het incidentModel: daar staat hij al